hnn_core/pyramidal.py

```diff
@@ -303,29 +318,40 @@ def _get_soma_props(self, pos):
             'name': 'L2Pyr',
         }
 
-    def geom(self, p_dend):
-        """The geometry."""
-        soma = self.soma
-        dend = self.list_dend
+    def secs(self):
+        """The geometry of the default sections in the Neuron."""
+        sec_pts = {
```

Comment:
Could we put in a reference to where the units of these values are defined? Belongs in Params, maybe?

Reply:
let's keep it here for now but I added info about units